### PR TITLE
Create a unified README for ndb samples

### PR DESCRIPTION
**Pull request by elibixby**
overview/ and transactions/  READMEs' are nearly identical, we should consolidate similar information, and consider adding more detailed readmes to the subdirectories.

### CODE REVIEW

File: datastore/ndb/README.md [hunk 1]
```diff
@@ -1,5 +1,58 @@
...
+   gcloud auth login
+   gcloud config set project <your-app-id>
+   ```
+   You don't need a valid app-id to run locally, but will need a valid id to deploy below.
+
+1. Clone this repo.
+
+   ```
+   git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
+   cd python-docs-samples/datastore/ndb/<sub-directory>
```
Comment (tmatsuo) [inline datastore/ndb/README.md]: Actually you can not run datastore/ndb/modeling. You can only run the tests.

Comment (elibixby) [inline datastore/ndb/README.md]: Should we make it runnable, or update the README to make it clear that not all sub-directories can be run (leaning towards the former).

**Comment by tmatsuo** [comment]
The reason why the travis build is failing is that the pull request is coming from another repo. I recommend that you create a branch in this repo and create a pull request from there.

File: datastore/ndb/README.md (hunk 1 above)
Comment (tmatsuo) [inline datastore/ndb/README.md]: Making the modeling subdir runnable in a meaningful way is somewhat cumbersome. Personally I'm OK only with unittests.

**Comment by elibixby** [comment]
ahhh travis being weird again... will do

**Pull request closed without merging**